File: algorithms/660. Remove 9/solution.cc
class Solution {
public:
    int newInteger(int n) {
        int base9 = 0, p = 1;
        while ( n > 0 ) {
            base9 += n % 9 * p;
            n /= 9;
            p *= 10;
        }
        return base9;
    }
};

File: algorithms/660. Remove 9/solution.py
class Solution(object):
    def newInteger(self, n):
        """
        :type n: int
        :rtype: int
        """
        base9 = 0
        p = 1
        while n > 0:
            base9 += (n % 9) * p
            p *= 10
            n /= 9
        return base9
